providers/gov/nih/normalizer.py

```diff
@@ -7,9 +9,24 @@
 
 
 def filter_nil(obj):
-        if isinstance(obj, dict) and obj.get('@http://www.w3.org/2001/XMLSchema-instance:nil'):
-            return None
-        return obj
+    if isinstance(obj, dict) and obj.get('@http://www.w3.org/2001/XMLSchema-instance:nil'):
+        return None
+    return obj
+
+
+def format_org_address(doc):
+    org_city = doc.get('ORG_CITY', '')
+    org_state = doc.get('ORG_STATE', '')
+    org_zipcode = doc.get('ORG_ZIPCODE', '')
+    org_country = doc.get('ORG_COUNTRY', '')
+    if not org_city and not org_state and not org_zipcode and not org_country:
```

Comment:
Don't bother changing it but `not any((org_city, org_state, org_zipcode, org_country)):` is a touch cleaner